fortran/utils_ppser.f90

```diff
@@ -84,9 +87,9 @@ SUBROUTINE ppser_initialize(directory, prefix, mode, prefix_ref, mpi_rank, rprec
 
   ! Get data size
   intvalue = 0
-  realvalue = 0.
+  realvalue = 4 ! default real type value 
   ppser_intlength = INT(SIZE(TRANSFER(intvalue, buffer)))
-  ppser_reallength = INT(SIZE(TRANSFER(realvalue, buffer)))
+  ppser_reallength = realtype
```

Comment:
I think there was a bug in my branch, what happen if real type is not present (it is an optional argument)